jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/LambdaBuilder.kt

```diff
@@ -82,6 +82,10 @@ abstract class LambdaBuilder {
                 override fun onTextAvailable(event: ProcessEvent, outputType: Key<*>) {
                     // TODO: We should find a way to show the output of this in the UI
                     LOG.info { event.text }
+
+                    if (ApplicationManager.getApplication().isUnitTestMode) {
+                        println("SAM CLI: ${event.text}")
```

Comment:
do we want to keep this?

Reply:
It is currently the only way to debug the unit tests. When we switch over to a pre task we can see if we can remove this.